x-pack/plugins/ml/public/application/routing/use_refresh.ts

```diff
@@ -26,13 +26,17 @@ export interface Refresh {
 export const useRefresh = () => {
   const timefilter = useTimefilter();
 
+  const getTimRange = () => {
```

Comment:
typo Tim

Reply:
fixed in 95171b2a377488c53244ec1c4f9e31a38b366c4c